### PR TITLE
Rework of City Screen: new current buildings list + misc changes

### PR DESCRIPTION
**Pull request by vegeta1k95**
City Screen has been partially reworked:

1) Button "Show drilldown stats" removed. Drilldown stats (buildings) are inserted into right city info panel as expander tabs.
image

2) Categories which were inserted have been given different look (Civ 5 style):
image

Each item shows: icon, name, output stats and specialists slots.
Specialists slots are now showed as filled, if enough of corresponding specialists of that type is active in the city.
image

3) "Sell" building button has been moved to bottom-right tooltip which is called by clicking on a building in the list:
image

Also misc. changes were made:
- Now all tiles in CityScreen show yields, this is default Civ 5 behaviour and is very helpful when considering which tiles to buy next (without it, you need to click manually every tile to see yields)
- Resources icons in CityScreen now are not dimmed.

### CODE REVIEW

**Comment by vegeta1k95** [comment]
> Now all tiles in CityScreen show yields, this is default Civ 5 behaviour and is very helpful when considering which tiles to buy next (without it, you need to click manually every tile to see yields)

Okay, I figured out this is the case in Civ 5 only when yields are enabled on a global map. If global map yields are disabled then in City Screen only city tiles show yields, not those outside.

I will implement this in the next commit

**Comment by yairm210** [comment]
We can show all yields of tiles in a 3-tile radius even though that's not what Civ V does, since it helps

**Comment by yairm210** [comment]
Anyway, checked this out myself, looks AMAZING!!!

**Comment by yairm210** [comment]
Wow what an upgrade
I never would have thought to do this myself

**Comment by yairm210** [comment]
Only one thing is not 100% perfect and that's that the specialist-providing buildings should always have the specialists colored, so the user knows what they provide, and not only if they're actually in use

**Comment by Ouaz** [comment]
Interesting improvement... but:

1) you removed the "Show All/Collapse all" toggle button. :(

Then again, we have to scroll the whole list (which can be very long) when wanting to select something at the bottom (or collapse each section one by one)

The button was added by this commit: https://github.com/yairm210/Unciv/pull/6441, based on request https://github.com/yairm210/Unciv/issues/6373

2) So, the detailed stats per city are gone? It allowed to check a policy or belief was applied to a particular city. Now, we just have global stats per city in the City Overview screen.

**Comment by jmuchemb** [comment]
I fail to see how this PR improves things. The main thing I was looking at when clicking on "Show drilldown stats" were stat calculations and I don't see them anymore.

On the other side, the other boxes have almost no value for me. In most cases:
- looking at what I can build is enough to deduce what to build
- I almost only sell buildings when razing a city

File: core/src/com/unciv/ui/cityscreen/ConstructionInfoTable.kt [hunk 1]
```diff
@@ -78,11 +89,36 @@ class ConstructionInfoTable(val cityScreen: CityScreen): Table() {
...
-            val link = (construction as? IRulesetObject)?.makeLink() ?: return
-            if (link.isEmpty()) return
-            touchable = Touchable.enabled
-            onClick {
-                UncivGame.Current.pushScreen(CivilopediaScreen(city.getRuleset(), link = link))
+            // Show sell button if construction is a currently sellable building
+            if (construction is Building && cityConstructions.isBuilt(construction.name)
+                    && construction.isSellable()) {
+                val sellAmount = cityScreen.city.getGoldForSellingBuilding(construction.name)
+                val sellText = "Sell [$sellAmount] " + Fonts.gold
```
Comment (Ouaz) [inline core/src/com/unciv/ui/cityscreen/ConstructionInfoTable.kt:96]: `Sell` This line is not translated in other languages.

sell

**Comment by yairm210** [comment]
I didn't even notice the stat drilldowns weren't there, you're right, that's the reason that panel originally existed in the first place
The stat drilldown expander tabs should be before the building tabs

**Comment by yairm210** [comment]
All right, they're back

**Pull request merged**